camera_parameters:
  matrix: [1.289061962331979885e+03,0.000000000000000000e+00,3.644093594041585789e+02,
          0.000000000000000000e+00,1.282227445659755631e+03,2.094685991831462957e+02,
          0.000000000000000000e+00,0.000000000000000000e+00,1.000000000000000000e+00]
  distortion: [1.245543717844518200e-01,-2.597473972245496476e+00,1.995496499458032907e-03,1.289794849173935345e-02,2.000510879977276701e+01]
opencv_detector:
  frontal_face_xml_path: /home/kevin/catkin_ws/src/camera/src/haarcascade_frontalface_alt.xml
  eye_xml_path: /home/kevin/catkin_ws/src/camera/src/haarcascade_eye_tree_eyeglasses.xml
  yaw_controller:
    kp: 0.02
    kd: 0.01
    ki: 0.0
  pitch_controller:
    kp: 0.02
    kd: 0.01
    ki: 0.0
